R/armax.R

```diff
@@ -0,0 +1,53 @@
+#' ARMAX Transfer Function models
+#'
+#' ARMAX linear transfer functions with a single input and single output
+#' series. Can be used as a general Unit Hydrograph transfer function, defined
+#' by Auto-Regressive and Moving Average coefficients.
+#'
+#' The transfer function used here, with input \var{u} and output \var{x} is:
+#' \deqn{x[t] = a_1 x[t-1] + \ldots + a_n x[t-n] + }{ x[t] = a[1] x[t-1] + ...
+#' + a[n] x[t-n] + b[0] u[t-d] + ... + b[m] u[t-m-d]}\deqn{ b_0 u[t-\delta] +
+#' \ldots + b_m u[t-m-\delta]}{ x[t] = a[1] x[t-1] + ... + a[n] x[t-n] + b[0]
+#' u[t-d] + ... + b[m] u[t-m-d]}
+#'
+#' and the \emph{order} is denoted \eqn{(n, m)}, with delay \eqn{\delta}{d}.
+#'
+#' @aliases armax armax.sim ssg.armax normalise.armax
+#' @param U input time series.
```

Comment:
I was expecting this to be an annotation on armax.sim. I noticed you said there were issues with the armax functions. Why did you end up putting this as its own file?